download_analysis_data.py

```diff
@@ -103,4 +43,10 @@ def download_worker(queue, outDir):
     except OSError:
         pass
 
-    download_analysis_files(args.outDir)
+    urlBase = 'https://web.lcrc.anl.gov/public/e3sm/diagnostics'
+    analysisFileList = pkg_resources.resource_string(
+            'mpas_analysis', 'obs/analysis_input_files').decode('utf-8')
+
+    # remove any empty strings from the list
+    analysisFileList = list(filter(None, analysisFileList.split('\n')))
```

Comment:
@milenaveneziani, I had to add this line because there's an extra blank line in `analysis_input_files` (which is fine) and as a result the code tries to download `https://web.lcrc.anl.gov/public/e3sm/diagnostics` (which, of course, fails).